src/pysdmx/model/message.py

```diff
@@ -30,3 +35,24 @@ class ActionType(Enum):
     Replace = "replace"
     Delete = "delete"
     Information = "information"
+
+
+class Header(Struct, frozen=True, kw_only=True):
+    """Header for the SDMX-ML file."""
+
+    id: str = "test"
+    test: str = "true"
+    prepared: datetime = datetime.strptime("2021-01-01", "%Y-%m-%d")
+    sender: str = "Unknown"
+    receiver: str = "Not_Supplied"
+    source: str = "PySDMX"
+    dataset_action: str = ActionType.Information.value
+
+    def __post_init__(self) -> None:
```

Comment:
This check is no longer necessary if test becomes a boolean?

Reply:
Removing this check as set test to boolean.